src/plugins/region_map/public/choropleth_layer.js

```diff
@@ -46,7 +47,7 @@ import {
 const EMPTY_STYLE = {
   weight: 1,
   opacity: 0.6,
-  color: 'rgb(200,200,200)',
+  color: euiThemeVars.euiColorMediumShade,
```

Comment:
This is kind of an interesting case. From the sample screenshots, I'd also make the leap to guess that it's used for both the outline and fill color (but would need a deeper dive into the code to verify), and is intended to be a neutral default. However, given it's usage, I _think_ something like `ouiPaletteColorBlind()[1]` (or a seed color from one of the quantitative palettes, like `ouiPaletteGray()`) may be more appropriate for cohesion with other visualizations. 

To help us decide, @BSFishy can you provide more info about how/when this `EMPTY_STYLE` is used? And confirm whether the `color` prop is only used for strokes or also for fill?

Reply:
This style is simply passed to leaflet, and I can now confirm it is only used as the stroke color: https://leafletjs.com/reference.html#path-color

Reply:
Thanks for the context, that's helpful.